src/ig/IGExporter.ts

```diff
@@ -949,6 +953,11 @@ export class IGExporter {
       if (pathResourceDirectories) predefinedResourcePaths.push(...pathResourceDirectories);
     }
     const deeplyNestedFiles: string[] = [];
+    const configuredBinaryResources = (this.config.resources ?? []).filter(
+      resource =>
+        resource.reference?.reference.startsWith('Binary/') &&
```

Comment:
We probably need to be defensive about both references here to avoid crashing if the nested reference is missing.
```suggestion
        resource.reference?.reference?.startsWith('Binary/') &&
```

Reply:
Whoops! and oops I forgot to accept the the suggestion so the fix is in [e16e997](https://github.com/FHIR/sushi/pull/1418/commits/e16e997417025854c6a4b99d7aeaa9fc5acbc314).